app/code/core/Mage/Paypal/Model/Express/Checkout.php

```diff
@@ -142,7 +142,7 @@ class Mage_Paypal_Model_Express_Checkout
     /**
      * Order
      *
-     * @var Mage_Sales_Model_QuoteMage_Sales_Model_Quote
+     * @var Mage_Sales_Model_Quote
```

Comment:
See lines 611 to 637.
```suggestion
     * @var Mage_Sales_Model_Order
```